src/applications/veteran-id-card/veteran-id-card-entry.jsx

```diff
@@ -1,12 +1,16 @@
+import React from 'react';
 import '../../platform/polyfills';
 import './sass/veteran-id-card.scss';
 
 import startApp from '../../platform/startup';
+import VeteranIDCard from './containers/VeteranIDCard';
+import Main from './containers/Main';
 
-import routes from './routes';
 import reducer from './reducers';
 
 startApp({
   reducer,
-  routes
+  component: (<VeteranIDCard>
```

Comment:
@jbalboni not sure how I feel about this- I think I could safely just render `Main` in `VeteranIDCard`- doesn't look like a reusable component.